SVGEngine.mm

```diff
@@ -214,6 +219,34 @@ @interface SVGAttributeSet () {
         return path;
 }
 
+CF_RETURNS_RETAINED CGPathRef svgParser::readCircleTag()
+{
+    NSCAssert(strcasecmp((char*)xmlTextReaderConstName(_xmlReader), "circle") == 0,
+              @"Not on a <circle>");
+    CGPoint const center = {
+        readFloatAttribute(@"cx"),     readFloatAttribute(@"cy")
+    };
+    float radius = readFloatAttribute(@"r");
+    CGMutablePathRef circle = CGPathCreateMutable();
+    CGPathAddArc(circle, NULL, center.x, center.y, radius, 0.0, (360 * M_PI), NO);
```

Comment:
CGPathAddArc takes an angle in radians, not degrees. (2π)

But we could also just use CGPathAddEllipseInRect I think (rx == ry). That way the code for ellipse & circle could be shared.